### PR TITLE
Siralex214/update about

### PR DESCRIPTION
**Pull request by siralex214**

### CODE REVIEW

**Review by AntoineKM — COMMENTED** [review]
Younès B. - Lead Developer
Marah M. - Junior CM
Inès M. - Senior CM
Alois G. - Lead CM
Lucas B. - Admin & Designer
Killian M. - Admin & Product Owner
Jeremy B. - Admin & Developer
Antoine K. - Admin & Developer

**Review by AntoineKM — APPROVED** [review]

**Comment by AntoineKM** [comment]
@siralex214 we remove the draft mode ?

File: src/data/members.ts [hunk 1]
```diff
@@ -22,25 +24,55 @@ const members: Member[] = [
...
+  },
+  {
+    firstname: "Mareh",
+    lastname: "Mannaa",
+    job: "main.roles.comunity-manager",
+    states: "main.status.junior",
+    thumbnail_url: "/static/images/members/mareh.jpg",
+    external_url: "https://www.linkedin.com/in/mareh-mannaa-0a9088231/",
+    joined_at: "2022/11/02",
   },
```
Comment (jerembdn) [inline src/data/members.ts:76]: You forgot Alois :(

**Review by jerembdn — CHANGES_REQUESTED** [review]

File: src/data/members.ts (hunk 1 above)
Comment (jerembdn) [reply src/data/members.ts:76]: ahok

**Review by jerembdn — APPROVED** [review]

**Comment by jerembdn** [comment]
Can you rebase on the base branch ?

**Pull request merged**